test/ut/sdk/test_model_speedup.py

```diff
@@ -349,7 +349,7 @@ def test_speedup_integration_small(self):
         self.speedup_integration(model_list)
 
     def test_speedup_integration_big(self):
-        model_list = ['vgg11', 'vgg16', 'resnet34', 'squeezenet1_1',
+        model_list = ['vgg11', 'resnet34', 'squeezenet1_1',
```

Comment:
How about skip the `speedup_integration_test` on macos and windows rather than remove the vgg model?

Reply:
it also breaks the latest ubuntu, so I think maybe remove it is a better choice, does vgg16 have some special structure that we need to test?

Reply:
No big deal, we can remove it as long as we still keep vgg11.